src/altinn-app-frontend/src/components/base/MapComponent.test.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from 'react';
+import { render as rtlRender, screen } from '@testing-library/react';
+import configureStore from 'redux-mock-store';
+import { Provider } from 'react-redux';
+
+import type { IComponentProps } from 'src/components';
+import { MapComponent } from './MapComponent';
+import type { IMapComponentProps } from './MapComponent';
+
+import { setupServer, handlers } from '../../../testUtils';
+
+const server = setupServer(...handlers);
+
+beforeAll(() => server.listen());
+afterEach(() => server.resetHandlers());
+afterAll(() => server.close());
```

Comment:
Theres no server traffic in this component/test, so this server mock is unnescessary and can be removed

```suggestion
```